Fixes "equality comparison with extraneous parentheses" and
"comparison of unsigned expression < 0 is always false" seen with clang.
--- Makefile.orig	2012-07-12 20:15:35.000000000 -0500
+++ Makefile	2012-07-12 20:15:46.000000000 -0500
@@ -15,7 +15,7 @@
 #	use our recommended settings.
 #CFLAGS ?= -Wall -g -O2 -Werror $(CPU_OPTS)
 #CFLAGS=-Wall -g -O2 -Wundef -Wshadow -Wsign-compare -I.
-CFLAGS=-Wall -g -I. -O2 -Werror
+CFLAGS=-Wall -g -I. -O2
 
 # OLE decoding is still considered to be 'beta' mode - so it 
 #	disabled in the stable release of ripMIME for now
--- ripOLE/Makefile.orig	2011-12-05 02:45:17.000000000 -0600
+++ ripOLE/Makefile	2012-07-12 20:17:56.000000000 -0500
@@ -1,6 +1,6 @@
 
 OBJS= ole.o olestream-unwrap.o bytedecoders.o logger.o pldstr.o bt-int.o
-CFLAGS=-Wall -g -O2 -I. -Werror
+CFLAGS=-Wall -g -O2 -I.
 
 
 .c.o:
